File: java/org.financer.client.javafx/src/main/java/org/financer/client/javafx/dialogs/FinancerExceptionDialog.java
package org.financer.client.javafx.dialogs;

import javafx.scene.layout.StackPane;
import org.financer.client.local.Application;

public class FinancerExceptionDialog extends FinancerAlert {
    public FinancerExceptionDialog(String header, Exception exception) {
        super(Application.MessageType.ERROR, header, null);
    }

    public FinancerExceptionDialog(StackPane container, String header, Exception exception) {
        super(container, Application.MessageType.ERROR, header, null);
    }
}

File: java/org.financer.client.javafx/src/main/kotlin/org/financer/client/javafx/dialogs/FinancerExceptionDialog.kt
package org.financer.client.javafx.dialogs

import javafx.scene.layout.StackPane
import org.financer.client.local.Application

class FinancerExceptionDialog : FinancerAlert {
    constructor(header: String?, exception: Exception?) : super(Application.MessageType.ERROR, header, null) {}
    constructor(container: StackPane?, header: String?, exception: Exception?) : super(container, Application.MessageType.ERROR, header, null) {}
}